src/plugins/telemetryTable/pluginSpec.js

```diff
@@ -183,10 +183,11 @@ describe("the plugin", () => {
 
         it("Renders a column for every item in telemetry metadata", () => {
             let headers = element.querySelectorAll('span.c-telemetry-table__headers__label');
-            expect(headers.length).toBe(3);
-            expect(headers[0].innerText).toBe('Time');
-            expect(headers[1].innerText).toBe('Some attribute');
-            expect(headers[2].innerText).toBe('Another attribute');
+            expect(headers.length).toBe(4);
+            expect(headers[0].innerText).toBe('Name');
+            expect(headers[1].innerText).toBe('Time');
+            expect(headers[2].innerText).toBe('Some attribute');
+            expect(headers[3].innerText).toBe('Another attribute');
         });
 
```

Comment:
We should add a test case here to ensure that a name column is automatically added.

Reply:
It does line 187 checks for Name column, telemetry metadata does not have it.